faq_bot/bot/bot.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from dff.messengers.telegram import PollingTelegramInterface, TelegramMessenger
+from dff.pipeline import Pipeline
+from dff.script.core.context import Context
+
+from .script import script
+from .model import find_similar_questions
+
+
+def question_processor(ctx: Context):
```

Comment:
Can we add a little be more comments, that   `question_processor` run first and into context simularities

Reply:
https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/7b9d6521717f1d9073a1245b8e163d1b67c41618